libraries/chain/db_market.cpp

```diff
@@ -31,7 +31,17 @@
 
 #include <fc/uint128.hpp>
 
-namespace graphene { namespace chain {
+namespace graphene { namespace chain { namespace detail {
+
+   fc::uint128 calculate_percent(const share_type& value, uint16_t percent)
```

Comment:
Perhaps change return value to `uint64_t` so you don't always have to call `to_uint64`?